orchestration/src/main/java/com/sap/ai/sdk/orchestration/spring/OrchestrationChatModel.java

```diff
@@ -17,13 +18,22 @@
 import org.springframework.ai.chat.model.ChatResponse;
 import org.springframework.ai.chat.prompt.Prompt;
 
-/** Spring AI integration for the orchestration service. */
+/**
+ * Spring AI integration for the orchestration service.
+ *
+ * @since 1.2.0
+ */
+@Beta
```

Comment:
(Major)

Why are you destabilizing public API?
We can only allow that when PO gave approval.

Reply:
Sorry, made a mistake.